exporter/sumologicexporter/sender_test.go

```diff
@@ -69,7 +69,7 @@ func prepareSenderTest(t *testing.T, compression configcompression.Type, cb []fu
 	case configcompression.TypeDeflate:
 		cfg.ClientConfig.Compression = configcompression.TypeDeflate
 	default:
-		cfg.CompressEncoding = configcompression.TypeGzip
+		cfg.ClientConfig.Compression = configcompression.TypeGzip
```

Comment:
Kept Gzip since this is the default value (safest way?). 

Another way I can think of is:
1. to remove default since switch statement covers all cases: empty string ("") and [other types of compressions](https://pkg.go.dev/go.opentelemetry.io/collector/config/configcompression#Type)

Reply:
We would like to avoid it, as it may require migration on customer side. For this option it shouldn't be a big issue, but technically changind default is an another breaking change.

We probably will introduce it (as changing `zstd` to default) in near future 😅